build-logic/dependency-modules/src/main/kotlin/gradlebuild/modules/extension/ExternalModulesExtension.kt

```diff
@@ -50,6 +50,7 @@ abstract class ExternalModulesExtension {
     val commonsLang = "commons-lang:commons-lang"
     val commonsLang3 = "org.apache.commons:commons-lang3"
     val commonsMath = "org.apache.commons:commons-math3"
+    val configurationCacheReport = "org.gradle.gradlebuild:configuration-cache-report"
```

Comment:
We need to agree on groupId/artifactId for this artifact. I don't think `gradlebuild` is the right groupId since this is not something for building Gradle, but actually something which is part of the distribution.

Reply:
I am also not sure if we should publish the artifact on `mavenCentral`, given that nobody else apart from us should use it.

Reply:
I suggest we use something like `org.gradle.buildtool.internal` for now to discourage public consumption. And for the same reason I don't thin we should publish the artifact on `mavenCentral`.

Reply:
It's now `org.gradle.buildtool.internal`.